expr/ct.go

```diff
@@ -47,6 +47,13 @@ const (
 	CtKeyAVGPKT     CtKey = unix.NFT_CT_AVGPKT
 	CtKeyZONE       CtKey = unix.NFT_CT_ZONE
 	CtKeyEVENTMASK  CtKey = unix.NFT_CT_EVENTMASK
+
+	// https://sources.debian.org/src//nftables/0.9.8-3/src/ct.c/?hl=39#L39
+	CtStatBitINVALID     uint32 = 1
```

Comment:
This still reads CtStatBit, but should read CtStateBit (note the `e`)

Reply:
sorry about that. I now added the `e`